deploy/openshift/manifests/00-contrail-09-manager.yaml

```diff
@@ -40,19 +40,19 @@ spec:
           analyticsStatisticsTTL: 2
           containers:
             - name: analyticsapi
-              image: <CONTRAIL_REGISTRY>/contrail-analytics-api:<CONTRAIL_VERSION>-ubi
+              image: <CONTRAIL_REGISTRY>/contrail-analytics-api:<CONTRAIL_VERSION>
```

Comment:
Why are you deleting UBI here?

Reply:
Ok I get it :)

Reply:
Because now user will have to specify full tag in config file for script generating manifests. Before we had to take tag without suffix, because some images were with and some without suffix. Now it'll be more consistent :)

Reply:
Although, these manifests are now used only for development. Customer should use release manifests already generated with release images